.gitignore

```diff
@@ -48,7 +48,7 @@ _ReSharper*/
 *.vssscc
 $tf*/
 *.dll
-publish/*
+artifacts/
 PropertyChangedAnalyzers.Demo/*
 .vs/*
 /PropertyChangedAnalyzers.Benchmarks/BenchmarkDotNet.Artifacts/*
```

Comment:
looks like some copy paste noise here ^ (not your fault)

Reply:
Ah, didn't even see it. Can I delete?

Reply:
yes, delete everything with `PropertyChangedAnalyzers` not sure how that got there.

Reply:
Want to update to the current gitignore.io for VS?

Reply:
Sounds good, maybe push that to master and rebase this PR?

Reply:
Done.